NuRadioReco/modules/ARA/hardwareResponseIncorporator.py

```diff
@@ -46,9 +47,18 @@ def run(self, evt, station, det, sim_to_data=False):
 
             else:
                 trace_before_system_fft = np.zeros_like(trace_fft)
-                trace_before_system_fft[np.abs(system_response['gain']) > 0] = trace_fft[np.abs(system_response['gain']) > 0] / (system_response['gain'] * system_response['phase'])[np.abs(system_response['gain']) > 0]
+                trace_before_system_fft[np.abs(system_response['gain']) > 0] = (
+                    trace_fft[np.abs(system_response['gain']) > 0] /
+                    (system_response['gain'] * system_response['phase'])[np.abs(system_response['gain']) > 0]
+                )
                 channel.set_frequency_spectrum(trace_before_system_fft, channel.get_sampling_rate())
 
+        if not sim_to_data:
+            # Subtraces the cable delay. For `sim_to_data=True`, the cable delay is added
+            # in the efieldToVoltageConverter or with the channelCableDelayAdder
+            # (if efieldToVoltageConverterPerEfield was used).
+            signal_processing.add_cable_delay(station, det, sim_to_data=False, logger=self.logger)
+
```

Comment:
This is something I added to make this module (the ARA hardwareResponseIncorporator) more similar to the once for ARIANNA and RNO-G. I could remove it again to not change the functionality of the module?

Reply:
I don't have strong feelings. I don't think a lot of people are using this actively. I would be ok, to change it.